src/asynchronous/server.rs

```diff
@@ -132,20 +132,20 @@ impl Server {
 
                 let incoming = UnixIncoming::new(unix_listener);
 
-                self.do_start(listenfd, incoming).await
+                self.do_start(incoming).await
             }
             #[cfg(target_os = "linux")]
             Some(Domain::Vsock) => {
                 let incoming = unsafe { VsockListener::from_raw_fd(listenfd).incoming() };
-                self.do_start(listenfd, incoming).await
+                self.do_start(incoming).await
             }
             _ => Err(Error::Others(
                 "Domain is not set or not supported".to_string(),
             )),
         }
     }
 
-    pub async fn do_start<I, S>(&mut self, listenfd: RawFd, mut incoming: I) -> Result<()>
+    pub async fn do_start<I, S>(&mut self, mut incoming: I) -> Result<()>
```

Comment:
Though the signature of this pub method has been changed, I have no plan to bump the major version of `ttrpc`. Because this method has hardly been used, even if this incompatibility is used, we can bear it.